src/genie/libs/parser/nxos/show_logging.py

```diff
@@ -74,3 +74,62 @@ def cli(self, include='', output=None):
                 continue
 
         return parsed_dict
+
+class ShowLoggingLevelSchema(MetaParser):
+    '''Schema for:
+        * 'show logging level'
+        * 'show logging level {name}'
+    '''
+
+    schema = { 'facility':
+                { Any():
+                    { 'def_severity': int,
+                      'curr_severity': int }
```

Comment:
Key names should be a little closer to what they are in the output. Try 
```suggestion
                    { 'default_severity': int,
                      'current_session_severity': int }
```